wger/gym/views/gym.py

```diff
@@ -56,6 +57,7 @@
     WgerDeleteMixin,
     WgerMultiplePermissionRequiredMixin)
 from wger.utils.helpers import password_generator
+import json
```

Comment:
`import json`  should be moved to the top of the script ```Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.``` [reference](https://www.python.org/dev/peps/pep-0008/#imports)